sound/soc/sof/sof-of-dev.c

```diff
@@ -23,6 +24,14 @@ static struct sof_dev_desc sof_of_imx8qxp_desc = {
 	.nocodec_tplg_filename = "sof-imx8-nocodec.tplg",
 	.ops = &sof_imx8x_ops,
 };
+
+static struct sof_dev_desc sof_of_imx8qm_desc = {
+	.default_fw_path = "imx/sof",
+	.default_tplg_path = "imx/sof-tplg",
+	.default_fw_filename = "sof-imx8.ri",
+	.nocodec_tplg_filename = "sof-imx8-nocodec.tplg",
+	.ops = &sof_imx8_ops,
+};
```

Comment:
I find this a bit confusing, earlier you mentioned that ' i.MX8QXP which is from i.MX8X", so is this really an IMX8 case?
Are you going to use the same firmware/topology for both?

Reply:
We will not have the same firmware for i.MX8 (i.MX8QM) / i.MX8X (i.MX8QXP).
Github diff helps with confusion here:

Looking at the code we have:
* for i.MX8 (i.MX8QM) ->  .default_fw_filename = "sof-imx8.ri",
* for i.MX8X (i.MXQXP) -> .default_fw_filename = "sof-imx8x.ri"

Nocodec topology is the same for i.MX8/i.MX8X. Other can be different but it doesn't matter at this point the names
of topology files will be read from device tree.

imx8 naming it is a little bit confusing, but after a while you will get used to it.

Simple tree of i.MX8 boards looks like this:

```
imx8
├── imx8
│   ├── imx8dm (has DSP)
│   └── imx8qm (has DSP)
├── imx8m
│   ├── im8mm
│   ├── im8mn
│   ├── im8mp (has DSP)
│   └── im8mq
└── imx8x
    ├── imx8dxl
    └── imx8qxp (has DSP)
```